onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -49,6 +49,9 @@
 #ifdef USE_CUDA
 #include "core/providers/cuda/cuda_provider_factory.h"
 #endif
+#ifdef USE_TRT
```

Comment:
did you test whether the python bindings work with TRT?
I would expect the onnxruntime_python.cmake file needs to be modified to build with the appropriate dependencies.
please take a look at cuda and mkldnn examples.

Reply:
python binding for TRT is working now.